sigsci_site_manager/backup.py

```diff
@@ -31,7 +31,10 @@ def get_custom_signals(api):
 def get_signal_rules(api):
     keys = ['enabled', 'signal', 'groupOperator', 'conditions', 'reason']
     data = api.get_signal_rules()
-    return filter_data(data['data'], keys)
+    try:
+        return filter_data(data['data'], keys)
+    except KeyError:
+        return {}, []
```

Comment:
Add logic to `filter_data` so that it can't throw a `KeyError`, which will cover all instances of using `filter_data`

Reply:
Fixed